recipe/post-link-ucc.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+cat << EOF >> $PREFIX/.messages.txt
+
+On Linux, Open MPI is built with UCC support but it is disabled by default.
+To enable it, first install UCC (conda install -c conda-forge ucc).
+Afterwards, set the environment variables
+OMPI_MCA_coll_ucc_enable=1
+before launching your MPI processes.
+Equivalently, you can set the MCA parameters in the command line:
+mpiexec --mca coll_ucc_enable 1 ...
```

Comment:
I think this is correct, thank you for checking 👍 

I have mixed feeling in this. While following the MCA convention we can just do
```
OMPI_MCA_coll=ucc
```
or
```
mpiexec --mca coll ucc
```
to enable the UCC support, this is surprisingly not taught in, e.g., UCC's documentation:
https://github.com/openucx/ucc/blob/master/docs/user_guide.md#getting-started
so I guess we should follow what the UCC team suggests, given lack of `coll` documentation in Open MPI... (cc: @jsquyres for vis)